tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/service/impl/CurriculumMembershipServiceTest.java

```diff
@@ -107,4 +115,119 @@ void shouldThrowExceptionWhenPmNotFound() {
     assertThrows(NoSuchElementException.class,
         () -> cmService.save(dto));
   }
+
+  @Test
+  void shouldPatchCmDto() throws MethodArgumentNotValidException, NoSuchMethodException {
+    CurriculumMembershipDTO dto = createDto(CURRICULUM_ID, PM_UUID, START_DATE, END_DATE);
+    dto.setId(CURRICULUM_ID);
```

Comment:
Can we use the constant "CM_ID" here please? I think it's confusing to set CURRICULUM_ID to a curriculum membership dto id.